### ISSUE TITLE
A couple of things about 'Fundraising/New Individual Raise'

### ISSUE THREAD

**Issue by jonlac** [aragon/issues#77]
1st: When you try to set the 'Share Price' and instead of using the arrows you type it in and then press 'Enter', it sends Aragon to sleep.
![share price](https://user-images.githubusercontent.com/29462786/27443276-cfa94c5a-5772-11e7-8bcc-30603e42e441.png)

2nd: The 'Share Price' number format leads to figures overlaying.
![share price 2](https://user-images.githubusercontent.com/29462786/27443337-070f9aaa-5773-11e7-9e04-c89e3e698447.png)

3rd: (Suggestion) It would be good to ban past dates in 'Round closes' interval to avoid leading to an instant closing.
![round closes](https://user-images.githubusercontent.com/29462786/27443517-9df48f70-5773-11e7-8732-fbf0c5e877f8.png)

And that's it. 
Best regards,

@lacasito

**Comment by izqui** [comment]
Thanks for the report, paging in @harshjv

**Comment by kevinhughes27** [comment]
I'll add my feedback to this issue as well. Here are a few things I noticed on this page.

In this section:

![image](https://user-images.githubusercontent.com/1965489/27447188-19ddb268-574e-11e7-9028-255949fee2e8.png)

It took me a second to realize that USD did make sense to use here since its a unit of value most people are familiar with. It would be helpful to show the math and the total amount of ETH required underneath in the UI. eg:

100 * 0.003 ETH = 0.3 ETH (100 USD)

I actually didn't send the right amount of ETH to the contract the first time and the error message wasn't super helpful - it took me a second to figure out what was going wrong. I don't know exactly how this would work but linking to an invoice page would be pretty cool for this.

Another thing on this page that would be helpful is a tooltip explaining the exact consequences of the close date.

**Comment by harshjv** [comment]
@jonlac Hey, can you confirm that you didn't change anything in "Round Closes" in the first issue? Also, regardless of "Round Closes", did you see the MetaMask popup to confirm/sign transactions/data? Because I can't reproduce this issue.

**Comment by jonlac** [comment]
No, I didn't. It got stuck before I could change it.  And if I clicked out it recovered but repeating the typing/enter got it stuck again. And I didn't remember to see either the MetaMask popping up, but I just did it again to check it out and it did pop up (without giving it a date nor a name), and look what happened once confirmed (thrice) and approved after the voting. 
![funraising](https://user-images.githubusercontent.com/29462786/27549924-0108711c-5a9e-11e7-80ab-95fcd6f92c9c.png)
I also tried typing in the 'Amount' in a new payment and it also got stuck. 
![accounting](https://user-images.githubusercontent.com/29462786/27550020-57d93788-5a9e-11e7-9222-c375f70bef1a.png)
[I'm using the desktop version in Windows 10]. Let me know if I can try something to clarify the issue. :)

**Comment by harshjv** [comment]
@jonlac Thanks a lot mate for the info. I'll try to reproduce this issue with given details on a Windows PC. I'll ping you if I need something. :)
